modules/ai/assets/js/editor/components/prompt-history/parts/modal-item-action-button.js

```diff
@@ -0,0 +1,22 @@
+import { IconButton } from '@elementor/ui';
+import Tooltip from '../../tooltip';
+
+const ActionButton = ( { tooltipTitle, ...props } ) => {
+	return (
+		<Tooltip title={ tooltipTitle } placement="top">
+			<IconButton
+				type="button"
+				size="small"
+				disableRipple={ true }
```

Comment:
I don't need you need the { true } attribute with react.
You can just disableRipple and it automatically true.

https://stackoverflow.com/a/53326274/1928647

Reply:
I also prefer this way but kept it to save consistency with the rest of the AI codebase. 
Looking forward we can easily change it using an eslint rule.